guides/source/6_0_release_notes.md

```diff
@@ -257,10 +257,63 @@ Please refer to the [Changelog][action-view] for detailed changes.
 
 ### Removals
 
+*   Remove deprecated `image_alt` helper.
+    ([Commit](https://github.com/rails/rails/commit/60c8a03c8d1e45e48fcb1055ba4c49ed3d5ff78f))
+
+*   Remove RecordTagHelper.
```

Comment:
I wonder we should have this as an entry as the gem was already extracted out. If we want to have it, then we can expand a bit ~

```
Remove empty `RecordTagHelper` module from which the functionality was already moved to `record_tag_helper` gem.
```

Reply:
I wondered the same, and I like your recommendation, I'll make that change.